source/docs/software/wpilib-tools/shuffleboard/custom-widgets/creating-plugins.rst

```diff
@@ -83,6 +83,8 @@ specifying what to copy. The group and description parameters to specifiy what t
 In the body, the ``from`` field specifies from where the file is to be copied from, followed by the ``into`` field specifying the destination to where the file needs to be copied.
 Finally, the ``include`` field ensures the ``.jar`` extention is also copied.
 
+After deploying to Shuffleboard, open Shuffleboard, and use Ctrl + L to open the plugins menu, and select clear cache. This is necessary since Shuffleboard automatically caches the plugin when it is loaded, and clearing the cache updates the cached version to the file in the plugins folder.
```

Comment:
The cache is just so Shuffleboard knows where the plugin JAR is so it can be loaded automatically at the next startup. It doesn't cache specific versions. See [here](https://github.com/wpilibsuite/shuffleboard/blob/60076f619a67a4a43bb7f7854123e1bedf212bcf/app/src/main/java/edu/wpi/first/shuffleboard/app/PluginPaneController.java#L162) and [here](https://github.com/wpilibsuite/shuffleboard/blob/60076f619a67a4a43bb7f7854123e1bedf212bcf/app/src/main/java/edu/wpi/first/shuffleboard/app/plugin/PluginCache.java)